ckanext/unaids/helpers.py

```diff
@@ -162,3 +162,16 @@ def get_google_analytics_id():
     if not from_env:
         return toolkit.config.get('ckan.google_analytics_id', None)
     return from_env
+
+
+def is_an_estimates_dataset(word):
```

Comment:
This solution is way too big overkill for the job. Creating the helper is excellent idea but it could be much simpler:
```
def is_an_estimates_dataset(dataset_type_name):
    return 'estimates' in dataset_type_name
```

Reply:
Fixed